store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9227,28 +9227,28 @@ TODO: delete them permanently from here
+<attr id="2090" name="zimbraMailAdminSieveScriptBefore" type="string" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited" since="8.7.8">
 <desc>sieve script defined by admin (not able to edit and view from the end user) applied before the end user filter rule</desc>
 </attr>
 
-<attr id="2091" name="zimbraMailAdminSieveScriptAfter" type="string" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="accountCosDomainInherited,domainInherited" since="8.7.8">
+<attr id="2091" name="zimbraMailAdminSieveScriptAfter" type="string" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited" since="8.7.8">
 <desc>sieve script defined by admin (not able to edit and view from the end user) applied after the end user filter rule</desc>
 </attr>
 
-<attr id="2092" name="zimbraMailAdminOutgoingSieveScriptBefore" type="string" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="accountCosDomainInherited,domainInherited" since="8.7.8">
+<attr id="2092" name="zimbraMailAdminOutgoingSieveScriptBefore" type="string" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited" since="8.7.8">
 <desc>outgoing sieve script defined by admin (not able to edit and view from the end user) applied before the end user filter rule</desc>
 </attr>
 
-<attr id="2093" name="zimbraMailAdminOutgoingSieveScriptAfter" type="string" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="accountCosDomainInherited,domainInherited" since="8.7.8">
+<attr id="2093" name="zimbraMailAdminOutgoingSieveScriptAfter" type="string" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited" since="8.7.8">
 <desc>outgoing sieve script defined by admin (not able to edit and view from the end user) applied after the end user filter rule</desc>
 </attr>
 
-<attr id="2094" name="zimbraSieveRejectEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="accountCosDomainInherited,domainInherited" requiresRestart="mailbox" since="8.7.8">
+<attr id="2094" name="zimbraSieveRejectEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.7.8">
 <globalConfigValue>TRUE</globalConfigValue>
 <desc>Whether to enable the Sieve "reject" action defined in RFC 5429.</desc>
 </attr>
 
-<attr id="2095" name="zimbraMailSieveNotifyActionRFCCompliant" type="boolean" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="accountCosDomainInherited,domainInherited" requiresRestart="mailbox" since="8.7.8">
+<attr id="2095" name="zimbraMailSieveNotifyActionRFCCompliant" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.7.8">
```

Comment:
Can you please set zimbraMailSieveNotifyActionRFCCompliant=TRUE on server level restart the mailbox then set zimbraMailSieveNotifyActionRFCCompliant=FALSE and do not restart the mailbox, will behavior remain same? 
Also

Set zimbraMailSieveNotifyActionRFCCompliant=FALSE on server level restart the mailbox then set zimbraMailSieveNotifyActionRFCCompliant=TRUE and do not restart the mailbox, will behavior remain same?